### ISSUE TITLE
View Fasttext Model layers and validation loss

### ISSUE THREAD

**Issue by abhi111q** [facebookresearch/fastText#1057]
How can anyone find the layers used in fasttext and train the data so that validation loss is visible along with every epoch simultaneously ??

How many and what are the layers used in fasttext ??

**Comment by Celebio** [comment]
Hi @abhi111q ,
There is only one layer in the fastText model. For every input `x`, we compute `h = Ax` that is the unique hidden layer, and gives the representation in the latent space. Then we compute the output logits `y = Bh`.  `A` is the "input matrix", representing the word vectors. `B` is the "output matrix", representing the classifier. 

Unfortunately there is no way to see validation loss automatically at each epoch. There isn't any checkpoint system implemented.

I hope this clarifies a little bit.

Best regards,
Onur

**Comment by iedmrc** [comment]
Is there any workaround to achieve this?
